### PR TITLE
Package docs cleanup

### PR DESCRIPTION
**Pull request by jsta**
See discussion at #228

### CODE REVIEW

**Comment by jread-usgs** [comment]
take a look at how this does against `test-nml.R line 48`?

**Comment by jsta** [comment]
Ok, I figured it out. Now passing an unnamed list of length 1 should throw the appropriate error:

> 'arg_list must be a named list'

 instead of:

> 'arg_name should be a character'

File: R/convert_sim_var.R [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #' @param \dots an expression to convert variable based on other variables or offsets
 #' @param unit the units for the new variable
 #' @param longname the longname for the new variable
+#' @param overwrite logical overwrite existing variable data?
```
Comment (lawinslow) [inline R/convert_sim_var.R]: You can drop the question mark. Just say "overwriting existing variable after conversion" or something like that.

File: R/dot_compare_to_field.R [hunk 2]
```diff
@@ -20,7 +20,7 @@
 	obs_metric <- vector('numeric', length = length(un_dates))
 	dates <- as.POSIXct(rep(NA, length = length(un_dates)))
 	
-	for (j in 1:length(un_dates)){
+	for (j in seq_len(length(un_dates))){
```
Comment (lawinslow) [inline R/dot_compare_to_field.R]: Nice. I should use this more. It solves the 1:0 problem I've run into in the past.

File: R/plot_compare_stage.R [hunk 3]
```diff
@@ -1,10 +1,9 @@
-#'@title Plot simulated and observed stage data
+#'Plot simulated and observed stage data
```
Comment (lawinslow) [inline R/plot_compare_stage.R]: Is this suggested syntax, or is it just a fancy way to do the documentation? As in, is there advantage to dropping the @title tag?

File: R/read_field_obs.R [hunk 4]
```diff
@@ -1,64 +1,65 @@
-#' @title read in field data into a data.frame
```
Comment (lawinslow) [inline R/read_field_obs.R]: Hard to tell what happened in this document. Line ending change?

File: DESCRIPTION [hunk 5]
```diff
@@ -2,12 +2,13 @@ Package: glmtools
 Type: Package
 Title: glmtools
 Version: 0.14.6
```
Comment (lawinslow) [inline DESCRIPTION]: Want to increment the minor version for the next release?

File: R/plot_compare_stage.R (hunk 3 above)
Comment (jsta) [inline R/plot_compare_stage.R]: Dropping the title tag makes it so the text is included in the `title` as well as the `description` to satisfy `devtools::check`. Alternatively, I could have added a `@description` tag but I think this way is cleaner.

File: R/read_field_obs.R (hunk 4 above)
Comment (jsta) [inline R/read_field_obs.R]: This is just removing the title tag per my response above. Not sure why the diff doesn't render the new line without the tag but its there in the source.

**Comment by lawinslow** [comment]
Awesome. Thanks for the cleanup @jsta!

**Pull request merged**